com.ktechpit.orion.yaml

```
@@ -0,0 +1,195 @@
+app-id: com.ktechpit.orion
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.15-22.08'
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: orion
+finish-args:
+  - --device=all
+  - --env=INSTALL_TYPE=flatpak
+  - --filesystem=/tmp:rw
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-pictures:rw
+  - --filesystem=xdg-run/gvfsd
+  - --filesystem=xdg-videos
+  - --filesystem=xdg-videos:rw
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --talk-name=org.freedesktop.FileManager1
+  - --own-name=org.mpris.MediaPlayer2.mpv.*
+  - --talk-name=org.gnome.SessionManager
```

```suggestion
  - --device=all
  - --env=INSTALL_TYPE=flatpak
  - --filesystem=/tmp
  - --filesystem=xdg-download
  - --filesystem=xdg-pictures
  - --filesystem=xdg-run/gvfsd
  - --filesystem=xdg-videos
  - --share=ipc
  - --share=network
  - --socket=fallback-x11
  - --socket=pulseaudio
  - --socket=wayland
  - --talk-name=org.freedesktop.FileManager1
  - --own-name=org.mpris.MediaPlayer2.mpv.*
  - --talk-name=org.gnome.SessionManager
```


It's still a very broad set of permissions

---

done, Thanks